.changeset/moody-pandas-fall.md

```diff
@@ -1,56 +0,0 @@
----
-"@fluidframework/tree": minor
-"fluid-framework": minor
----
----
-"section": tree
----
-
-There are now `@alpha` APIs for schema evolution which support adding optional fields to object node types without a staged rollout.
-
```

Comment:
Can't make a suggestion, but the change I want to make is to add a title to this changelist, just a new sentence above the existing text, "New alpha APIs for schema evolution".

Reply:
That will require a separate change to the already generated release notes. Might make sense to just do that in its own PR. Looks like this PR has a conflict from a removed package. I'm going to regenerate it. Maybe make that change in main and merge it first?

Reply:
https://github.com/microsoft/FluidFramework/pull/23464